train/arch-and-scaling-template.slurm

```diff
@@ -177,5 +177,10 @@ export CMD=" \
 
 echo $CMD
 
+# We create the folder where the logs and codecarbon will be stored.
+mkdir -p $LOGS_PATH
+# Uncomment if you use codecarbon
+# mkdir -p $CODECARBON_PATH
+
```

Comment:
```suggestion
```

Sorry, I didn't follow closely. This is not needed, since CC auto-creates the path if it's not there already:

```
    Path(output_dir).mkdir(parents=True, exist_ok=True)
```